### DISCUSSION TITLE
v0.7.1-586 > 2 commits available Invalid

### DISCUSSION THREAD

**Discussion by RobbieRw** [Arksine/moonraker#692] [General]
Hi Guys

Iv turned on my FLSUN V400 this morning to find i have invalid next to moonraker. Iv tried to hard and soft recover but giving me lots of errors. 

<img width="657" alt="Screenshot 2023-07-27 092135" src="https://github.com/Arksine/moonraker/assets/36843841/1da8e820-c5c8-4e05-96ee-04a438e34cde">
<img width="586" alt="Screenshot 2023-07-27 093144" src="https://github.com/Arksine/moonraker/assets/36843841/66a3c96c-17a8-4e23-a6b8-57ecad522c2c">
<img width="824" alt="Screenshot 2023-07-27 093254" src="https://github.com/Arksine/moonraker/assets/36843841/b9c5d3c5-f630-4624-bd1b-c0f2bf2d36ee">

Any ideas on how i can resolve this please?

**Comment by engelsheidt** [comment]
You can try to rename the moonraker directory in your host machine to something like moonraker2 and then git clone the most recent version, make sure to name the newly downloaded directory exactly the same as your original one, then reboot the host and check mainsail.

**Comment by gv2710** [comment]
Таже проблема, не могу решить..
